features/admin.users.v1/components/users-list.tsx

```diff
@@ -257,18 +260,38 @@ export const UsersList: React.FunctionComponent<UsersListProps> = (props: UsersL
 
     /**
      * Returns a locked icon if the account is locked.
+     * Returns a cross icon if the account is disabled.
      *
      * @param user - each admin user belonging to a row of the table.
      * @returns the locked icon.
      */
-    const resolveAccountLockStatus = (user: UserBasicInterface): ReactNode => {
+    const resolveAccountStatus = (user: UserBasicInterface): ReactNode => {
         const accountLocked: boolean = user[userConfig.userProfileSchema]?.accountLocked === "true" ||
             user[userConfig.userProfileSchema]?.accountLocked === true;
+        const accountDisabled: boolean = user[userConfig.userProfileSchema]?.accountDisabled === "true" ||
+            user[userConfig.userProfileSchema]?.accountDisabled === true;
         const accountLockedReason: string = user[userConfig.userProfileSchema]?.lockedReason;
 
         const accountLockedReasonContent: string = ACCOUNT_LOCK_REASON_MAP[accountLockedReason]
             ?? ACCOUNT_LOCK_REASON_MAP["DEFAULT"];
 
+        if (accountDisabled) {
+            return (
+                <Popup
```

Comment:
Don't we have a oxygen UI icon for this?

Reply:
We do not have a matching icon in oxygen-ui (solid icons) that is why this icon was used